app/blog/20161004-how-to-update-wordpress.static.md

```diff
@@ -0,0 +1,53 @@
+---
+title: How to update WordPress
+author:  Jenny Knuth
+posted: 20161004
+summary:
+  Steps to take to update and test a theme update in WordPress.
+_options:
+  layout: app/layouts/blog.static.hbs
+---
+
+## For the Developer:
```

Comment:
Perhaps a summary of the top of main tasks?

- Update and Test on local environment
- Check in and Test on staging
- Deploy and Verify on live

This could provide context for the descriptions below.